1.12.0/csi-operator-1.12.0-RC1.yaml

```diff
@@ -490,22 +490,6 @@ status:
   conditions: []
   storedVersions: []
 
----
-
-apiVersion: storage.k8s.io/v1beta1
-kind: CSIDriver
-metadata:
-  name: cstor.csi.openebs.io
-spec:
-  # Supports persistent and ephemeral inline volumes.
-  volumeLifecycleModes:
-  - Persistent
-  - Ephemeral
-  # To determine at runtime which mode a volume uses, pod info and its
-  # "csi.storage.k8s.io/ephemeral" entry are needed.
-  podInfoOnMount: true
-  attachRequired: true
-
 ---
```

Comment:
Q: In the starting of the file do we need to specify it supported only in so and so OS? As mentioned in other places?
```
# only with Ubuntu 16.04 and CentOS hosts, due to dependencies on
--
  | # linux distros. |   | # kernel components of iSCSI protocol.  For other ubuntu flavours and
  |   |   | # linux distros, this needs to be modified.
  |   |   | #
  |   |   | # Instructions to modify:
  |   |   | # Ubuntu-18.04 and above: https://github.com/openebs/cstor-csi/blob/master/deploy/iscsiadm-ubuntu-18.04-and-above-deps.yaml
  |   |   | # SUSE Linux Enterprise Server 12:  https://github.com/openebs/cstor-csi/blob/master/deploy/iscsiadm-suse-enterprise-server-12-deps.yaml
  |   |   | # SUSE Linux Enterprise Server 15:  https://github.com/openebs/cstor-csi/blob/master/deploy/iscsiadm-suse-enterprise-server-15-deps.yaml


```

Reply:
That's fine, i think its good to have this info there, in a single file.